purchase_request_to_requisition/wizard/purchase_request_line_make_purchase_requisition.py

```diff
@@ -77,8 +77,6 @@ def _get_requisition_line_search_domain(self, requisition, item):
                  item.product_uom_id.id or False),
                 ('account_analytic_id', '=',
                  item.line_id.analytic_account_id.id or False)]
-        if not item.product_id:
-            vals['name'] = item.name
```

Comment:
@jbeficent 
Will you want me to make changes in my PR, or you will just correct on your branch and just ignore this? Which is better?